test/parallel/test-runner-cli.js

```diff
@@ -269,10 +269,22 @@ const testFixtures = fixtures.path('test-runner');
 
 {
   // --test-shard option, first shard
+  const allShardsTestsFiles = [
+    'a.cjs',
+    'b.cjs',
+    'c.cjs',
+    'd.cjs',
+    'e.cjs',
+    'f.cjs',
+    'g.cjs',
+    'h.cjs',
+    'i.cjs',
+    'j.cjs',
+  ].map((file) => join(testFixtures, 'shards', file));
   const args = [
     '--test',
     '--test-shard=1/2',
-    join(testFixtures, 'shards/*.cjs'),
+    ...allShardsTestsFiles,
```

Comment:
Wouldn't this work?
```suggestion
    join(testFixtures, 'shards', '*'),
```

Reply:
it would but the whole reason we changed is that we want to backport the shards to version 20, see here for more:
- https://github.com/nodejs/node/pull/48639#issuecomment-1635421165

Reply:
@rluvaton right, but didn't it use Regex prior to globs? 
Maybe that only worked with `--test-name-pattern` though... would that be a valid solution?

Reply:
I don't think so, it was globs...